easybuild/easyconfigs/p/PASA/PASA-2.5.3-foss-2023a.eb

```
@@ -0,0 +1,53 @@
+easyblock = 'ConfigureMake'
+
+name = 'PASA'
+version = '2.5.3'
+
+homepage = 'https://github.com/PASApipeline/PASApipeline'
+description = """PASA, acronym for Program to Assemble Spliced Alignments (and pronounced 'pass-uh'),
+ is a eukaryotic genome annotation tool that exploits spliced alignments of expressed transcript
+ sequences to automatically model gene structures, and to maintain gene structure annotation consistent
+ with the most recently available experimental sequence data. PASA also identifies and classifies all
+ splicing variations supported by the transcript alignments."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+source_urls = ['https://github.com/PASApipeline/PASApipeline/archive/']
+sources = ['%(namelower)s-v%(version)s.tar.gz']
+checksums = ['9b2f6a301b73fd8c713af0977cf6e97f3d928988d6b79715ebe81e19b51152eb']
+
+dependencies = [
+    ('SQLite', '3.42.0'),
+    ('GMAP-GSNAP', '2023-04-20'),
+    ('pblat', '2.5.1'),
+    ('minimap2', '2.26'),
+    ('FASTA', '36.3.8i'),
+]
+
+buildininstalldir = True
+
+skipsteps = ['configure', 'install']
+
+unpack_options = '--strip-components=1'
+
+local_bins = [
+    '%(namelower)s',
+    'cdbfasta',
+    'cdbyank',
+    'cln2qual',
+    'mdust',
+    'psx',
+    'seqclean',
+    'seqclean.psx',
+    'slclust',
+    'trimpoly',
+]
+
```

This env variable needs to be set
```suggestion
modextrapaths = {
    'PASAHOME': '',
}
```